src/AppInstallerCLICore/Argument.cpp

```diff
@@ -198,6 +198,10 @@ namespace AppInstaller::CLI
         case Execution::Args::Type::IgnoreResumeLimit:
             return { type, "ignore-resume-limit"_liv, ArgTypeCategory::None };
 
+        // Font command
+        case Execution::Args::Type::FamilyName:
+            return { type, "family-name"_liv, ArgTypeCategory::None };

```

Comment:
Alternatively, why can't `--name` just be re-used for consistency with other winget commands? I think the spec in #4842 even calls it `--name` at [line 178](https://github.com/ryfu-msft/winget-cli/blob/91f932b7d9dc89e76076c209ac43630d4589be97/doc/specs/%23166%20-%20Font%20Support.md?plain=1#L178)

Reply:
I'm still noodling on whether we can reuse `--name`, I think for now I will leave it as `--family` so that its clear what we're referring to. I did make it positional so you can now just do `wingetdev fonts list 'Times New Roman'`